PKGBUILD

```diff
@@ -0,0 +1,40 @@
+# Maintainer: Yukari Chiba <i@0x7f.cc>
+
+pkgname=flite
+pkgver=2.2
+pkgrel=1
+pkgdesc="A lightweight speech synthesis engine"
+arch=(x86_64 aarch64 riscv64 loongarch64)
+url="http://www.festvox.org/flite/"
+license=(LicenseRef-Flite)
+depends=(alsa-lib)
+makedepends=(chrpath
+             git)
+source=(git+https://github.com/festvox/flite#tag=v$pkgver)
+sha256sums=('2f7873f3de7bb6d6a8368684fd2bacf771260e666966a75535a4c8206424071b')
+
+prepare() {
+  cd $pkgname
+  sed '/^#VOXES.*$/d; s/+//g; s/cmu_indic_lex/&\nVOXES = cmu_us_kal16 cmu_us_slt/' config/android.lv >config/eweos.lv
+  sed -i '/$(INSTALL) -m 755 $(BINDIR)\/flite_time $(DESTDIR)$(INSTALLBINDIR)/d' main/Makefile
+}
+
+build() {
+  cd $pkgname
+  ./configure \
+    --prefix=/usr \
+    --enable-shared \
+    --with-audio=alsa \
+    --with-vox=cmu_us_kal16 \
+    --with-langvox=eweos
+  make -j1
+}
+
+package() {
+  cd $pkgname
+  make DESTDIR="$pkgdir" install
+  install -D -m644 COPYING "$pkgdir"/usr/share/licenses/$pkgname/LICENSE
+
+# Fix rpath
+  chrpath -d "$pkgdir"/usr/bin/*
```

Comment:
Why we get broken rpath? Check whether this is really needed, we could add a TODO here if yes.